src/views/BrowseView/BrowseView.style.ts

```diff
@@ -30,12 +35,26 @@ export const StyledInfiniteVideoGrid = styled(InfiniteVideoGrid)`
 export const Container = styled.div`
   position: relative;
   padding-top: ${sizes(14)};
+  overflow-x: hidden;
 `
 export const IntersectionTarget = styled.div`
   min-height: 1px;
 `
+
 export const StyledBackgroundPattern = styled(BackgroundPattern)`
   position: absolute;
-  right: 0;
+  top: 73px;
+  right: 80px;
   z-index: ${zIndex.background};
+  transform: scale(1.3);
+  @media screen and (min-width: ${breakpoints.base}) {
```

Comment:
Since we assume we'll always be at minimum `breakpoints.base`, we actually don't need that media query. We can just include `display: none` as a default and it will later get overridden